src/ej/templates/jinja2/components/generic/page_header.jinja2

```diff
@@ -52,7 +52,7 @@
         </script>
     {% endblock %}
     <nav class="Header-topNav">
-        {% if request.path != '/start/' %}
+        {% if request.path != '/start/' and request.path != '/conversations/' and request.path != '/home/'%}
```

Comment:
Mais idiomático:  if request.path not in ['/start/', '/conversations/', '/home/']

Reply:
Arrumado, coloquei desta forma!